src/gui/editors/AutomationEditor.cpp

```diff
@@ -882,8 +882,8 @@ void AutomationEditor::mouseMoveEvent(QMouseEvent * mouseEvent )
 						? yCoordOfLevel(OUTVAL(it))
 						: yCoordOfLevel(INVAL(it));
 					float dy = m_draggedOutTangent
```

Comment:
(Non-blocking) You could just divide the result of the ternary instead of writing the division twice:

```cpp
float dy = (m_draggedOutTangent ? y - mouseEvent->y() : mouseEvent->y() - y) / m_y_delta;
```